.github/workflows/ci.yml

```diff
@@ -53,6 +53,7 @@ jobs:
     - name: Install WSExport
       run: |
         composer install
+        ./bin/console doctrine:migrations:migrate --env=test --no-interaction
```

Comment:
APP_ENV is already being set above in the `env` bit, it shouldn't be needed here.

Reply:
FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI